src/Form.js

```diff
@@ -2,20 +2,94 @@ import React from "react";
 import "./Form.css";
 
 class Form extends React.Component {
+  constructor() {
+    super();
+    this.state = {
+      userInput: "",
+      result: "",
+      func: "",
+      input: [],
+    };
```

Comment:
```suggestion
    this.state = {
      result: "",
      func: "",
      input: [],
    };
```
`input` and `userInput` are keeping track of the same thing.

The only difference is that `input` is the value of the input box converted into an array of numbers (instead of a string of number strings). You're really only using `input`. You can get rid of `userInput`, it's redundant.